flow/environment.py

```diff
@@ -41,6 +42,13 @@
 logger = logging.getLogger(__name__)
 
 
+@lru_cache(maxsize=1)
+def _cached_fqdn():
+    """We cache this since fetching the fully qualified domain name can be
+    slow on macOS."""
```

Comment:
@cbkerr Sorry, I probably wasn't clear about what I meant. The docstring you had before was mostly correct. I'm going to apply the following changes. You can see these [recommendations from pydocstyle with the corresponding codes](http://www.pydocstyle.org/en/stable/error_codes.html):
- Summary line (first line) shouldn't wrap. D205.
- Summary line should be in imperative mood (starts with a verb like "Gets" or "Computes"). D401.
- Blank line after the first line, then additional info. D205.
- Avoid first or second person words like we/you (not a pydocstyle recommendation, just my preference.)

We don't run pydocstyle right now but we will require pydocstyle to pass in the near future (#387).
```suggestion
    """Return the fully qualified domain name.
    
    This value is cached because fetching the fully qualified domain name can
    be slow on macOS.
    """
```